### PR TITLE
feat: Add json schema

### PR DESCRIPTION
**Pull request by Sukaato**
#1076 
Add json schema for configure release-it.json

### CODE REVIEW

File: schema/release.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "node_modules/release-it/schema/release",
```
Comment (webpro) [inline schema/release.json:3]: Why does this value contain `node_modules`? Is it supposed to reference itself?

File: schema/release.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "type": "object",
+      "additionalProperties": true
+    },
+    "git": {
+      "$comment": "Can be a boolean or git config object",
+      "if": {
+        "type": "object"
+      },
+      "then": {
+        "$ref": "node_modules/release-it/schema/git.json"
```
Comment (webpro) [inline schema/release.json:38]: Maybe this could be a relative reference to `"./git.json"`?

File: schema/release.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      "else": {
+        "type": "boolean",
+        "default": false
+      }
+    },
+    "npm": {
```
Comment (webpro) [inline schema/release.json:45]: Every plugin can be `false` (like `"git"`)

File: schema/test.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "file:///C:/Users/KANOMA1/Documents/projects/release-it/schema/release.json",
```
Comment (webpro) [inline schema/test.json:2]: Maybe this file should be omitted entirely?

**Review by webpro — COMMENTED** [review]
Thanks for the PR, @Sukaato. I have a few questions. Also: can this file be auto-generated somehow?

File: schema/release.json (hunk 2 above)
Comment (Sukaato) [reply schema/release.json]: I've just fixed all $ref path

File: schema/release.json (hunk 1 above)
Comment (Sukaato) [reply schema/release.json]: it's just a unique id, we can write we want, i've just change it for a better in latest commit

File: schema/release.json (hunk 3 above)
Comment (Sukaato) [reply schema/release.json:49]: fixed in last commit

File: schema/test.json (hunk 4 above)
Comment (Sukaato) [reply schema/test.json]: removed in last commit

**Comment by webpro** [comment]
:rocket: _This pull request is included in v17.4.0. See [Release 17.4.0](https://github.com/release-it/release-it/releases/tag/17.4.0) for release notes._

**Comment by webpro** [comment]
Thanks @Sukaato! This is the recommendation:

```json
{
  "$schema": "https://unpkg.com/release-it@17/schema/release-it.json"
}
```

I'll add it to the docs soon.

**Pull request merged**